### PR TITLE
[Sass 638] - Differentiate URL redirect link by actor

### PR DESCRIPTION
**Pull request by dnyuar**
### Description
Please include a brief summary of the change you've made and the issue that it fixes.

Add a link to the relevant story in Jira
[SASS-638] https://jira.tools.tax.service.gov.uk/browse/SASS-607

##### Before Reviewing
- [x]  Have you pulled the branch down?
- [x]  Have you assigned yourself to the PR?
- [x]  Have you moved the task to “in review” on JIRA?
- [x]  Have you checked to ensure all dependencies are up to date?

##### Whilst Reviewing
- [x]  Have you run the tests?
- [x]  Have you run the journey tests?
- [x]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [x]  Have you checked for merge conflicts?
- [x]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [x]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [x]  Have you run the tests?
- [x]  Have you run the journey tests?
- [x]  Have you addressed warnings where appropriate?

##### After PRs been Reviewed
- [ ]  Have you checked the PR Builder passes?
- [ ]  Have you checked for merge conflicts?
- [ ]  Have you checked code coverage isn’t lower than previously?

### CODE REVIEW

**Comment by robert-webb-hmrc** [comment]
Can this be rebased please? Journey Tests currently don't pass due to url changes that affect this repo also

**Comment by dnyuar** [comment]
branch been rebased

File: test/views/IVFailurePageSpec.scala [hunk 1]
```diff
@@ -69,7 +69,7 @@ class IVFailurePageSpec extends AnyWordSpec with Matchers with GuiceOneAppPerSui
   "Rendering the IVFailurePage in Welsh" should {
     "render correctly when the user is an individual" should {
```
Comment (robert-webb-hmrc) [inline test/views/IVFailurePageSpec.scala:70]: Both these cases are an individual user so should be (isAgent = false)

**Review by robert-webb-hmrc — CHANGES_REQUESTED** [review]
Please see above

File: test/controllers/SignOutControllerSpec.scala [hunk 2]
```diff
@@ -34,7 +34,7 @@ class SignOutControllerSpec extends UnitTest with DefaultAwaitTimeout {
 
         val request = FakeRequest("GET", "/sign-out")
 
-        val responseF = controller.signOut()(request)
+        val responseF = controller.signOut(false)(request)
```
Comment (robert-webb-hmrc) [inline test/controllers/SignOutControllerSpec.scala:37]: As mentioned on personal, could we consider having an agent case here?

File: test/views/helpers/BetaBarViewSpec.scala [hunk 3]
```diff
@@ -35,7 +35,8 @@ class BetaBarViewSpec extends ViewTest {
       "use appConfig.feedbackUrl in the beta banner link" which {
 
         implicit val appConfig: AppConfig = new MockAppConfig().config
-        lazy val view = betaBar()(fakeRequest, messages, appConfig)
+        implicit val isAgent: Boolean = false
+        lazy val view = betaBar(isAgent)(fakeRequest, messages, appConfig)
```
Comment (robert-webb-hmrc) [inline test/views/helpers/BetaBarViewSpec.scala:39]: As mentioned on personal, could we consider having an agent case here?

**Review by robert-webb-hmrc — CHANGES_REQUESTED** [review]

**Comment by robert-webb-hmrc** [comment]
Screenshot 2021-04-20 at 17 26 06

Can we address these warns please?

**Review by robert-webb-hmrc — APPROVED** [review]

**Pull request merged**